lib/active_model/serializer/attributes.rb

```diff
@@ -34,10 +34,13 @@ def inherited(base)
         #   class AdminAuthorSerializer < ActiveModel::Serializer
         #     attributes :id, :name, :recent_edits
         def attributes(*attrs)
+          options = attrs.last.class == Hash ? attrs.pop : {}
+          options = options.except(:key)
```

Comment:
good call.  I wonder if this could be supported via activesupport `with_options` and no code changes?